### PR TITLE
Test that events that are emitted during block finalization are included in the blocks

### PR DESCRIPTION
**Pull request by mrsmkl**
### Description

Just a simple test to check if these events are actually included in the blocks.

### Other changes

Also added some new events that are emitted when paying epoch rewards.

### Related issues

- Test for https://github.com/celo-org/celo-blockchain/pull/584

### CODE REVIEW

File: packages/celotool/src/e2e-tests/governance_tests.ts [hunk 1]
```diff
@@ -687,6 +687,15 @@ describe('governance tests', () => {
         }
       }
     })
+
+    it('should have emitted transfer events when paying epoch rewards', async () => {
```
Comment (asaj) [inline packages/celotool/src/e2e-tests/governance_tests.ts]: Can we test that the correct events are emitted, and also that the events are only emitted during epoch blocks?

Comment (mrsmkl) [reply packages/celotool/src/e2e-tests/governance_tests.ts]: Ok, adding more checks.

Comment (asaj) [reply packages/celotool/src/e2e-tests/governance_tests.ts]: Ah, didn't realize that we don't actually emit the main events that we'd want to record.

More specifically, can we check that the following events are emitted in finalizing the last block of the epoch, and that no events are emitted in finalizing any other block?

`TargetVotingYieldUpdated` (needs to be added to `EpochRewards.updateTargetVotingYield`)
`ValidatorEpochPaymentDistributed` (needs to be added to `Validators.distributeEpochPayementsFromSigner`)
`EpochRewardsDistributedToVoters` (needs to be added to `Election.distributeEpochRewards`)

And can you please add unit tests that check for these events as well?

Thanks!

**Review by asaj — APPROVED** [review]
Love it!

**Review by diminator — APPROVED** [review]
LGTM

**Pull request merged**